admin/client/components/EditFormHeader.js

```diff
@@ -46,6 +46,14 @@ var Header = React.createClass({
 		});
 	},
 
+	handleEscapeKey (event) {
+		const escapeKeyCode = 27;
+
+		if (event.which === escapeKeyCode) {
+			this.refs.searchField.getDOMNode().blur();
```

Comment:
We're using React .14, I believe this will throw a warning to use `ReactDOM.findDOMNode(ref)` instead. To see warnings, rebuild keystone's packages with `npm run build-dev`